openoa/utils/plot.py

```diff
@@ -1932,3 +1932,284 @@ def plot_wake_losses(
+                xlim = (
+                    vane_bins[valid_vane_indices[0]] - 1.0,
+                    vane_bins[valid_vane_indices[-1]] + 1.0,
+                )
+
+            axs[int(np.floor(i / N_col))][i % N_col].scatter(
+                vane_bins, power_values_vane_ws[i, :] / norm_factor, color=power_color_code
+            )
+            axs[int(np.floor(i / N_col))][i % N_col].plot(
+                vane_bins,
+                cos_curve(
+                    vane_bins,
+                    curve_fit_params_ws[i, 0] / norm_factor,
+                    curve_fit_params_ws[i, 1],
+                    curve_fit_params_ws[i, 2],
+                ),
+                color=curve_fit_color_code,
+                label="_nolabel_",
+            )
+            axs[int(np.floor(i / N_col))][i % N_col].plot(
+                2 * [curve_fit_params_ws[i, 1]],
+                [
+                    0.01 * np.floor(y_min / norm_factor / 0.01),
+                    0.01 * np.ceil(y_max / norm_factor / 0.01),
+                ],
+                color=curve_fit_color_code,
+                linestyle="--",
+                label=f"Max. Power Vane Angle = {round(curve_fit_params_ws[i,1],1)}$^\circ$",
+            )
+
+            axs[int(np.floor(i / N_col))][i % N_col].set_title(
+                f"{ws} m/s\nYaw Misalignment = {np.round(yaw_misalignment_ws[i],1)}$^\circ$"
+            )
+
+        axs[int(np.floor(i / N_col))][i % N_col].plot(
+            2 * [mean_vane_angle_ws[i]],
+            [
+                0.01 * np.floor(y_min / norm_factor / 0.01),
+                0.01 * np.ceil(y_max / norm_factor / 0.01),
+            ],
+            color=mean_vane_color_code,
+            linestyle="--",
+            label=f"Mean Vane Angle = {round(mean_vane_angle_ws[i],1)}$^\circ$",
+        )
+
+        axs[int(np.floor(i / N_col))][i % N_col].grid("on")
+
+        axs[int(np.floor(i / N_col))][i % N_col].legend(**legend_kwargs)
+
+        axs[int(np.floor(i / N_col))][i % N_col].set_ylabel(power_performance_label)
+
+        if ylim != (None, None):
+            axs[int(np.floor(i / N_col))][i % N_col].set_ylim(ylim)
+        else:
+            axs[int(np.floor(i / N_col))][i % N_col].set_ylim(
+                (
+                    0.1 * np.floor(y_min / norm_factor / 0.1),
+                    0.1 * np.ceil(y_max / norm_factor / 0.1),
+                )
+            )
+
+    for i in range(N_col):
+        axs[int(np.floor((len(ws_bins) - 1) / N_col))][i].set_xlabel("Wind Vane Angle ($^\circ$)")
+
+    mean_yaw_misalignment = np.mean(yaw_misalignment_ws)
+    if UQ:
+        yaw_misalignment_95CI = np.percentile(np.mean(yaw_misalignment_ws, 1), [2.5, 97.5])
+        fig.suptitle(
+            f"Turbine {turbine_id}, Yaw Misalignment = {np.round(mean_yaw_misalignment,1)}$^\circ$ [{np.round(yaw_misalignment_95CI[0],1)}$^\circ$, {np.round(yaw_misalignment_95CI[1],1)}$^\circ$]"
```

Comment:
Same here, particularly, the `np.round()` can be done when `yaw_misalignment_95CI` is created. You can also feel free to shorten the variable name if you like, even just dropping "yaw" or "misalignment"

Reply:
Updated